src/app/index.tsx

```diff
@@ -9,6 +12,8 @@ import { restoreWalletSession } from '@app/store/session-restore';
 import { App } from './app';
 import { setDebugOnGlobal } from './debug';
 
+bitcoin.initEccLib(ecc);
```

Comment:
Can you please explain what this is doing for my own curiosity?

Reply:
Not ideal, but we're using the bitcoinjs-lib because I didn't manage to get ledger working with btc-signer. 

This lib doesn't come with the elliptic curve library, so you have to initialise it separately. I added here to main page to do globally, though open for suggestion where else to initialise it. 

Ideally we remove the bitcoinjs-lib entirely.

Reply:
OK great, thank you! Maybe we should add a comment about it so the explanation is tracked?